components/Layout/Banner/LayoutBannerNewLearningPlatform.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <qiskit-banner>
+    Introducing&nbsp;
+    <a href="#">IBM Quantum Learning</a>, a new home for free quantum education
+    resources. Continue to use Learn on Qiskit until December 1st, 2023.
```

Comment:
Small change in the wording, so it is clear what  "Learn on Qiskit" means.
```suggestion
    resources. Continue to use qiskit.org/learn until December 1st, 2023.
```

Reply:
Done https://github.com/Qiskit/qiskit.org/pull/3411/commits/4015815a22642e2c0c6a2c519fd0f2d57bb3c4b4

Reply:
cc @JRussellHuffman @pandasa123